hooks/pre_prompt.py

```diff
@@ -5,6 +5,10 @@
 
 def get_latest_version(package_name):
     """Fetch the latest version of a package from PyPI."""
+
+    # pypi uses dashes, not underscores
+    package_name = package_name.replace('_', '-')
```

Comment:
On further investigation, it seems to work fine with underscores...

https://pypi.org/pypi/vivarium-public-health/json
https://pypi.org/pypi/vivarium_public_health/json

Reply:
If so then, I'd suggest removing this line. Less code is better!